app/controllers/people_controller.rb

```diff
@@ -0,0 +1,74 @@
+class PeopleController < ApplicationController
+  before_action :set_person, only: [:show, :edit, :update, :destroy]
+
+  # GET /people
+  # GET /people.json
+  def index
+    @people = Person.all
+  end
+
+  # GET /people/1
+  # GET /people/1.json
+  def show
+  end
+
+  # GET /people/new
+  def new
+    @person = Person.new
+  end
+
+  # GET /people/1/edit
+  def edit
+  end
+
+  # POST /people
+  # POST /people.json
+  def create
+    @person = Person.new(person_params)
+
+    respond_to do |format|
+      if @person.save
+        format.html { redirect_to @person, notice: 'Person was successfully created.' }
+        format.json { render :show, status: :created, location: @person }
+      else
+        format.html { render :new }
+        format.json { render json: @person.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /people/1
+  # PATCH/PUT /people/1.json
+  def update
+    respond_to do |format|
+      if @person.update(person_params)
+        format.html { redirect_to @person, notice: 'Person was successfully updated.' }
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.